graviti/utility/binary_file.py

```diff
@@ -25,8 +26,6 @@
 class FileBase(ReprMixin):
     """This class represents the file in a DataFrame."""
 
-    __slots__ = ("_key", "_extension", "_size")
```

Comment:
make sure your `__slots__` is valid.

Reply:
When inheriting from a class without __slots__, the __dict__ and __weakref__ attribute of the instances will always be accessible.
https://docs.python.org/zh-cn/3/reference/datamodel.html#notes-on-using-slots